tests/test_claim_filing_completeness.py

```
@@ -0,0 +1,79 @@
+    )
+    issues = enforce_article148_party_completeness(draft)
+    assert any("БИН" in issue for issue in issues)
+    assert any("место нахождения" in issue for issue in issues)
+    assert any("банковские реквизиты" in issue for issue in issues)
+
+
+def test_defendant_iin_bin_and_bank_are_not_hard_required_when_unknown():
+    draft = _draft(
+        claimant=[
+            "Иванов Иван Иванович, дата рождения 01.01.1990, ИИН 900101300001, "
+            "адрес: г. Алматы, ул. Абая, д. 10"
```

Ruff (0.16.2)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)